codecov.yml

```diff
@@ -15,5 +15,7 @@ coverage:
     # changes tells us if there are unexpected code coverage changes in other files
     # which were not changed by the diff
     changes: false
```

Comment:
nit: does this new way of generating coverage packages mean we still need this?  I think the answer is yes because of how codecov seems to rebase, but I'm not entirely sure.